app/controllers/search_controller.rb

```diff
@@ -1,59 +1,36 @@
 class SearchController < ApplicationController
-	before_filter :if_not_signed_in
+  before_filter :if_not_signed_in
```

Comment:
When you say improving before_filter in other controllers, do you mean this kind of change?